exercise/do-your-own-map.js

```diff
@@ -0,0 +1,113 @@
+/*
+## Do-Your-Own-Map
+
+### Instructions
+
+1) Create a `map` function that takes an array as first argument, a function as second,
+and that works like the method .map
+
+2) Create a `flatMap` function that takes an array as first argument, a function as second,
+and that works like the method .flatMap
+
+### Notions
+
+- Map
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+
+- FlatMap
+
+*/
+Array.prototype.map = undefined
+Array.prototype.flatMap = undefined
+
+///*/ // ⚡
+
+///*/// ⚡
+
+export const tests = []
+const t = (f) => tests.push(f)
+
+let arr1 = [10, -10, 20, -95, 86, 102, 35, 89, 110]
+let arr2 = [[10], -10, 20, -95, 86, [102], [35, 89], 110]
+let arr3 = [[5], [4], [-3], [20], [17], [-33], [-4], [18]]
```

Comment:
why `let` ?
also, those would be better in the `setup`

Reply:
force of habbit, did not think that one through. I swiched them to `const`